### PR TITLE
Greek jobs

### PR DESCRIPTION
**Pull request by athaks**
I just added 445 jobs in Greek

### CODE REVIEW

File: faker/providers/job/el_GR/__init__.py [hunk 1]
```diff
@@ -0,0 +1,453 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from .. import Provider as BaseProvider
+
+
+class Provider(BaseProvider):
+    jobs = [
+        'Αγγειοπλάστης',
```
Comment (fcurella) [inline faker/providers/job/el_GR/__init__.py]: do you have a source for this list? If so, could you include it as comment?

File: faker/providers/ssn/el_GR/__init__.py [hunk 2]
```diff
@@ -4,17 +4,31 @@
 
 class Provider(BaseProvider):
     """
-    A Faker provider for the Greek VAT IDs
+    A Faker provider for the Greek VAT IDs and Greek police IDs
```
Comment (fcurella) [inline faker/providers/ssn/el_GR/__init__.py]: this is out of scope for this PR. Could  you submit it as separate PR?

File: faker/providers/job/el_GR/__init__.py (hunk 1 above)
Comment (athaks) [reply faker/providers/job/el_GR/__init__.py]: there is no specific source, I was browsing the web gathering sample data to populate the job descriptions list

File: faker/providers/ssn/el_GR/__init__.py (hunk 2 above)
Comment (athaks) [reply faker/providers/ssn/el_GR/__init__.py]: apologies, this was meant to be my next PR but somehow slipped in this one

**Comment by athaks** [comment]
Added tests and removed accidentally slipped source file

**Comment by fcurella** [comment]
Thank you! ✨
